pkg/extender/extender_test.go

```diff
@@ -543,6 +548,58 @@ func multipleVolumeTest(t *testing.T) {
 		prioritizeResponse)
 }
 
+// Create a pod with 2 PVCs using the mock storage class.
+// Place the data for volume1 on nodes n1, n2.
+// Place the data for volume2 on nodes n2, n3.
+// Put the skip scoring label on volume2
+// Send requests with node n1, n2, n3, n4, n5
+// The filter response should not include node n3.
+// The prioritize response should assign priorities in the following order
+// n1 & n2 highest priority. n3 should have no priority
+func multipleVolumeSkipTest(t *testing.T) {
+	nodes := &v1.NodeList{}
+	nodes.Items = append(nodes.Items, *newNode("node1", "node1", "192.168.0.1", "rack1", "", ""))
+	nodes.Items = append(nodes.Items, *newNode("node2", "node2", "192.168.0.2", "rack2", "", ""))
+	nodes.Items = append(nodes.Items, *newNode("node3", "node3", "192.168.0.3", "rack3", "", ""))
+	nodes.Items = append(nodes.Items, *newNode("node4", "node4", "192.168.0.4", "rack1", "", ""))
+	nodes.Items = append(nodes.Items, *newNode("node5", "node5", "192.168.0.5", "rack2", "", ""))
+
+	if err := driver.CreateCluster(5, nodes); err != nil {
+		t.Fatalf("Error creating cluster: %v", err)
+	}
+
+	pod := newPod("doubleVolumeSkipPod", map[string]bool{"included-volume": false, "excluded-volume": true})
+
+	provNodes := []int{0, 1}
+	if err := driver.ProvisionVolume("included-volume", provNodes, 1, nil); err != nil {
+		t.Fatalf("Error provisioning volume: %v", err)
+	}
+	provNodes = []int{1, 2}
+	if err := driver.ProvisionVolume("excluded-volume", provNodes, 1, map[string]string{skipScoringLabel: "true"}); err != nil {
+		t.Fatalf("Error provisioning volume: %v", err)
+	}
+
+	filterResponse, err := sendFilterRequest(pod, nodes)
+	if err != nil {
+		t.Fatalf("Error sending filter request: %v", err)
+	}
+	verifyFilterResponse(t, nodes, []int{0, 1, 2, 3, 4}, filterResponse)
```

Comment:
filter response should not include 2 ?

Reply:
There is no change to the filtering logic. We are only skipping the volume in the prioritize request. Also even there the node will not be skipped but it will get a low score